src/Disks/IVolume.h

```diff
@@ -37,16 +70,17 @@ class IVolume : public Space
 
     /// Volume name from config
     const String & getName() const override { return name; }
+    virtual VolumeType getType() const = 0;
 
     /// Return biggest unreserved space across all disks
     UInt64 getMaxUnreservedFreeSpace() const;
 
-    Disks disks;
+    DiskPtr getDisk(size_t i = 0) const { return disks[i]; }
```

Comment:
Maybe we need something like `getMainDisk` or `getFirstDisk`? As I remember we discussed that we need something like that.

Reply:
Name `get*Disk` could be confusing in places where volume is supposed to have one disk, default argument solves this problem (for me, btw). Now it looks quite accurate: `getDisk()` where you suppose only on disk in volume (very many places) and `getDisk(i)` where it is needed (rare usage I believe). We can leave just `getDisk()` without any argument because it seems that we won't need the variant with argument at all.